packages/material-ui/src/SvgIcon/SvgIcon.js

```diff
@@ -117,7 +117,7 @@ SvgIcon.propTypes = {
   /**
    * The fontSize applied to the icon. Defaults to 24px, but can be configure to inherit font size.
    */
-  fontSize: PropTypes.oneOf(['inherit', 'default', 'small', 'large']),
+  fontSize: PropTypes.oneOf(['default', 'inherit', 'large', 'small']),
```

Comment:
Is the alphabetical sorting the expected behavior? I'm tempted to say that we might have logic in the order. Like from the most important values to the least or the "smallest" to the "biggest".

Reply:
Yes it's expected, I had issues with typescript not giving me unions in the same order which meant the output wasn't guaranteed to be the same every time.
So to work around that I sort everything in alphabetical order